media/js/firefox/new/portland-font-tracker.js

```diff
@@ -0,0 +1,4 @@
+// provided by fonts.com
+/* eslint-disable */
+eval(function(p,a,c,k,e,r){e=function(c){return c.toString(a)};if(!''.replace(/^/,String)){while(c--)r[e(c)]=k[c]||e(c);k=[function(e){return r[e]}];e=function(){return'\\w+'};c=1};while(c--)if(k[c])p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c]);return p}('4 7=g.b;4 2=3.p(\'r\');2.9=\'a/5\';2.c=\'d\';2.e=(\'6:\'==3.u.h?\'6:\':\'i:\')+\'//j.k.l/t/1.5?m=n&o=\'+7;(3.8(\'q\')[0]||3.8(\'s\')[0]).f(2);',31,31,'||mtiTracking|document|var|css|https|projectId|getElementsByTagName|type|text|MTIProjectId|rel|stylesheet|href|appendChild|window|protocol|http|fast|fonts|net|apiType|css|projectid|createElement|head|link|body||location'.split('|'),0,{}))
```

Comment:
This code should not be compressed/minified in the repository.

It is evaluating arbitrary third-party code that is essentially impossible to read. If we want to minify it, we should do it as part of the Bedrock build process.

Reply:
Unfortunately this is how the code is provided from the third party. I don't believe we have access to an unminified version. 😕

Reply:
Could we ask fonts.com for the unminified version? I would be hesitant to run this kind of code on our website, when run from a 3rd party.

Reply:
Hm, I'm going to try a different approach that loads external CSS as opposed to external JS. Will push a commit shortly.